supla-common/proto.h

```diff
@@ -499,24 +518,48 @@ extern char sproto_tag[SUPLA_TAG_SIZE];
 #define SUPLA_MFR_COMELIT 14
 #define SUPLA_MFR_POLIER 15
 
+// BIT map definition for TDS_SuplaRegisterDevice_*::Flags (32 bit)
 #define SUPLA_DEVICE_FLAG_CALCFG_ENTER_CFG_MODE 0x0010  // ver. >= 17
-#define SUPLA_DEVICE_FLAG_SLEEP_MODE_ENABLED 0x0020     // ver. >= 18
-
+#define SUPLA_DEVICE_FLAG_SLEEP_MODE_ENABLED    0x0020  // ver. >= 18
+#define SUPLA_DEVICE_FLAG_CALCFG_SET_TIME       0x0040  // ver. >= 20
+
+// BIT map definition for TDS_SuplaRegisterDevice_F::ConfigFields (64 bit)
+// type: TDeviceConfig_StatusLed
+#define SUPLA_DEVICE_CONFIG_FIELD_STATUS_LED            (1ULL << 0)  // v. >= 20
+// type: TDeviceConfig_ScreenBrightness
+#define SUPLA_DEVICE_CONFIG_FIELD_SCREEN_BRIGHTNESS     (1ULL << 1)  // v. >= 20
+// type: TDeviceConfig_ButtonVolume
+#define SUPLA_DEVICE_CONFIG_FIELD_BUTTON_VOLUME         (1ULL << 2)  // v. >= 20
+// type: TDeviceConfig_DisableLocalConfig
+#define SUPLA_DEVICE_CONFIG_FIELD_DISABLE_LOCAL_CONFIG  (1ULL << 3)  // v. >= 20
```

Comment:
Then shouldn't this be a device flag instead of a device configuration field?

Reply:
This field informs server that device accepts configuration of "button's volume" and "disabling local config".